src/components/learner-credit-management/data/utils.js

```diff
@@ -736,10 +736,17 @@ export const getAssignableCourseRuns = ({ courseRuns, subsidyExpirationDatetime,
   });
   // Sorts by the enrollBy date. If enrollBy is equivalent, sort by start.
   const sortedAssignableCourseRuns = assignableCourseRuns.sort((a, b) => {
-    if (a.enrollBy === b.enrollBy) {
-      return dayjs(a.start).unix() - dayjs(b.start).unix();
+    // Label relevant timestamps when to timestamp to milliseconds for the most granular sort
```

Comment:
nit/grammar: Is there another way to phrase this without "... when to timestamp to..."?

Reply:
lol yep sorry fixed 👍🏽